src/main/java/com/rakovets/course/java/core/practice/jcf_list/SchoolClass/comparator/compAge.java

```diff
@@ -0,0 +1,13 @@
+package com.rakovets.course.java.core.practice.jcf_list.SchoolClass.comparator;
+
+import com.rakovets.course.java.core.practice.jcf_list.SchoolClass.model.Student;
+
+import java.util.Comparator;
+
+    public class compAge implements Comparator<Student> {
+        @Override
+        public int compare(Student o1, Student o2) {
+            return Double.compare(o1.getAge(), o2.getAge());
+        }
+    }
+
```

Comment:
"Violation: There aren't correct indentations for inner code
Solution: Add correct indentations for inner code"